### DISCUSSION TITLE
Lock Accessory need help

### DISCUSSION THREAD

**Discussion by remobil** [RavenSystem/esp-homekit-devices#2335] [MEPLHAA Scripts]
I have an intercom, I soldered an esp8266 board into it, there is a temperature sensor, a doorbell button (a notification comes with a picture from the camera) and a door open button. Everything works except opening the lock. Help with the script. GPIO 4 Occupancy Sensor Accessory, GPIO 5 Thermometer Accessory, GPIO 14 Lock Accessory

{"c":{"io":[[[14],4,0],[[5],22,0],[[4],6,0]]},"a":[{"0":{"r":[[12,1]]},"1":{"r":[[12,0]],"b":[[12,1],[12,0]]},"t":4,"b":[],"s":1,"f0":[],"f1":[],"i":1.17},{"t":22,"b":[],"n":3,"g":5,"j":20,"z":8.5},{"t":6,"s":0,"i":10,"f1":[{"g":4,"t":1,"i":0,"p":0}]}]}

**Comment by pilot1981** [comment]
You can share your project?

**Comment by latraksk** [comment]
can you try
 
 {"c":{"io":[[[14],2],[[4,0],6]]},
 "a":[{
  "0":{"r":[[14,1]]},
  "1":{"r":[[14]],
  "b":[[0,1],[0]]},
  "t":4,
  "i":1.17
  },{
    "t":22,"n":3,"g":5,"j":20,"z":8.5
    },{
     "t":6,"i":10,"f1":[[4,1]]}]}

**Reply by remobil** [reply]
everything worked, this is what I need, thank you very much friend 
ps: https://glumb.github.io/haa-configurator/ this was much easier for me before
